env.go

```diff
@@ -59,3 +60,18 @@ func loadConfigFile() (*Config, error) {
 	err = json.NewDecoder(f).Decode(&config)
 	return &config, err
 }
+
+func writeConfigFile(r io.Reader) error {
+	dir := filepath.Join(os.Getenv("HOME"), ".tranor")
+	err := os.MkdirAll(dir, 0755)
```

Comment:
Why MkdirAll? Do we really want to attempt to create $HOME if it does not exist?